tests/playwright/sanity/includes/widgets/playing-cards.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect, test } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorPage from '../../../pages/editor-page';
+import PlayingCardContent from '../../../pages/widgets/playing-cards-page';
+test.describe( 'Playing Cards', () => {
+	test( 'it should display 1 playing card on adding the widget on default', async ( { page }, testInfo ) => {
+		const wpAdmin: WpAdminPage = new WpAdminPage( page, testInfo );
```

Comment:
Add `AAA` for each step 
Arrangement, Act, Asset

Reply:
Take a look at test structure in Nested Accordion or Mega Menu tests for some examples of this...